build_deploy_sc.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -exv
+
+IMAGE="quay.io/cloudservices/clowder"
+IMAGE_TAG=$(git rev-parse --short=8 HEAD)
+SECURITY_TAG="security-compliance"
+
+if [[ -z "$QUAY_USER" || -z "$QUAY_TOKEN" ]]; then
+    echo "QUAY_USER and QUAY_TOKEN must be set"
+    exit 1
+fi
+
+if [[ -z "$RH_REGISTRY_USER" || -z "$RH_REGISTRY_TOKEN" ]]; then
+    echo "RH_REGISTRY_USER and RH_REGISTRY_TOKEN  must be set"
+    exit 1
+fi
+
+BASE_TAG=`cat go.mod go.sum Dockerfile.base | sha256sum  | head -c 8`
+BASE_IMG=quay.io/cloudservices/clowder-base
+
+DOCKER_CONF="$PWD/.docker"
+mkdir -p "$DOCKER_CONF"
+docker --config="$DOCKER_CONF" login -u="$QUAY_USER" -p="$QUAY_TOKEN" quay.io
+docker --config="$DOCKER_CONF" login -u="$RH_REGISTRY_USER" -p="$RH_REGISTRY_TOKEN" registry.redhat.io
+
+RESPONSE=$( \
+        curl -Ls -H "Authorization: Bearer $QUAY_API_TOKEN" \
+        "https://quay.io/api/v1/repository/cloudservices/clowder-base/tag/?specificTag=$BASE_TAG" \
+    )
+
+echo "received HTTP response: $RESPONSE"
+
+# find all non-expired tags
+VALID_TAGS_LENGTH=$(echo $RESPONSE | jq '[ .tags[] | select(.end_ts == null) ] | length')
+
+if [[ "$VALID_TAGS_LENGTH" -eq 0 ]]; then
+    docker --config="$DOCKER_CONF" build -f Dockerfile.base . -t "${BASE_IMG}:${BASE_TAG}"
+	docker --config="$DOCKER_CONF" push "${BASE_IMG}:${BASE_TAG}"
+    docker --config="$DOCKER_CONF" tag "${BASE_IMG}:${BASE_TAG}" "${BASE_IMG}:${SECURITY_TAG}"
+    docker --config="$DOCKER_CONF" push "${BASE_IMG}:${SECURITY_TAG}"
+fi
+
+make update-version
+docker --config="$DOCKER_CONF" build  --build-arg BASE_IMAGE="${BASE_IMG}:${BASE_TAG}" -t "${IMAGE}:${IMAGE_TAG}" .
+docker --config="$DOCKER_CONF" push "${IMAGE}:${IMAGE_TAG}"
+docker --config="$DOCKER_CONF" tag "${IMAGE}:${IMAGE_TAG}" "${IMAGE}:${SECURITY_TAG}"
+docker --config="$DOCKER_CONF" push "${IMAGE}:${SECURITY_TAG}"
```

Comment:
@maknop, correct, but my worry is if there is a commit to main/master, there would be image created incorrectly flagged as a Security-Compliance image. But we can forgo the `build_deploy_sc.sh` and just be mindful of the `security-compliance` Git Commit Hash for the image tags.

Reply:
@maknop, thinking about it more, I agree with you. This is an over architected solution for something that is ultimately a non-issue since we can just look at the `security-compliance` Git Commit Hash for the image tag we need.

Reply:
`build_deploy_sc.sh` has been removed.